commit e899704b3db4af2ff228eb0d4dbbec05fb9a2448
Author: Hrvoje Senjan <hrvoje.senjan@gmail.com>
Date:   Mon Jun 10 09:26:37 2013 +0200

    Remove usage of (fake) Amarok-Mockup theme
    
    This is due to https://bugs.kde.org/show_bug.cgi?id=320855
    Amarok-Mockup is unused, as far i can see, and also makes Amarok crash with kdelibs master/4.11
    If it is still needed, or there is a better solution, i'll discard the review...
    
    From 6b85a2899a30db5acebee92182a67778499a8d9e Mon Sep 17 00:00:00 2001
    From: Hrvoje Senjan <hrvoje.senjan@gmail.com>
    Date: Sun, 9 Jun 2013 19:55:58 +0200
    Subject: [PATCH 1/1] Kill Amarok-Mockup
    
    BUG: 320855
    FIXED-IN: 2.8
    REVIEW: 110917

diff --git a/src/context/ContextView.cpp b/src/context/ContextView.cpp
index f61518f..152f8da 100644
--- a/src/context/ContextView.cpp
+++ b/src/context/ContextView.cpp
@@ -78,11 +78,6 @@ ContextView::ContextView( Plasma::Containment *cont, Plasma::Corona *corona, QWi
     p.setColor( QPalette::Base, c );
     setPalette( p );
 
-    PERF_LOG( "Accessing Plasma::Theme" );
-    // here we initialize all the Plasma paths to Amarok paths
-    Plasma::Theme::defaultTheme()->setUseGlobalSettings( false );
-    Theme::defaultTheme()->setThemeName( "Amarok-Mockup" );
-    PERF_LOG( "Access to Plasma::Theme complete" )
     contextScene()->setAppletMimeType( "text/x-amarokappletservicename" );
 
     cont->setPos( 0, 0 );
